<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>20</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(215, 215, 215)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Baner">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;</string>
    </property>
    <property name="text">
     <string>Kółko i krzyżyk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guzik_start">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START!!!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="guzik_kolko">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kółko</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="guzik_krzyzyk">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Krzyżyk</string>
    </property>
   </widget>
   <widget class="QLabel" name="wynik">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>22</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Wynik:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wynik_kolko">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kółko</string>
    </property>
   </widget>
   <widget class="QLabel" name="wynik_krzyzyk">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Krzyżyk</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="licznik_kolko">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 0,0);
text-align: center;</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="licznik_krzyzyk">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 0,0);
text-align: center;</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="wygrana">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>601</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zagraj od nowa !!!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="doilu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="text_doilu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Do ilu chcesz grać?</string>
    </property>
   </widget>
   <zorder>restart</zorder>
   <zorder>wygrana</zorder>
   <zorder>Baner</zorder>
   <zorder>guzik</zorder>
   <zorder>guzik_2</zorder>
   <zorder>guzik_3</zorder>
   <zorder>guzik_4</zorder>
   <zorder>guzik_5</zorder>
   <zorder>guzik_6</zorder>
   <zorder>guzik_7</zorder>
   <zorder>guzik_8</zorder>
   <zorder>guzik_9</zorder>
   <zorder>guzik_start</zorder>
   <zorder>guzik_kolko</zorder>
   <zorder>guzik_krzyzyk</zorder>
   <zorder>wynik</zorder>
   <zorder>wynik_kolko</zorder>
   <zorder>wynik_krzyzyk</zorder>
   <zorder>licznik_kolko</zorder>
   <zorder>licznik_krzyzyk</zorder>
   <zorder>doilu</zorder>
   <zorder>text_doilu</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
